libp2p/connmanager.nim

```diff
@@ -199,7 +218,10 @@ proc closeMuxerHolder(muxerHolder: MuxerHolder) {.async.} =
 
   await muxerHolder.muxer.close()
   if not(isNil(muxerHolder.handle)):
-    await muxerHolder.handle # TODO noraises?
+    try:
+      await muxerHolder.handle # TODO noraises?
+    except CatchableError as exc:
+      trace "Exception in close muxer handler", exc = exc.msg
```

Comment:
this should be a warning at least - since the convention is to _not_ raise, it should never happen that it raises